xsuite/src/world/fsworld.test.ts

```diff
@@ -223,6 +223,30 @@ test.concurrent("FSWorld.getAccountBalance", async () => {
   expect(await world.getAccountBalance(wallet)).toEqual(1234n);
 });
 
+test.concurrent("FSWorld.getAccountValue - Invalid key", async () => {
```

Comment:
"Invalid key": don't remember if I use an uppercase at this place in the other tests?

Reply:
After changing the `key` type to `BytesLike` this test doesn't make sense anymore because an invalid key error will be raised by our `bytesLikeToHex()` function.

I removed `invalid key` test in LS anf FS tests.